tests/functional/restart/28-execution-timeout.t

```diff
@@ -17,17 +17,12 @@
 #-------------------------------------------------------------------------------
 # Test restart with running task with execution timeout.
 . "$(dirname "$0")/test_header"
-set_test_number 4
+set_test_number 3
 install_workflow "${TEST_NAME_BASE}" "${TEST_NAME_BASE}"
 
 run_ok "${TEST_NAME_BASE}-validate" cylc validate "${WORKFLOW_NAME}"
 workflow_run_ok "${TEST_NAME_BASE}-run" cylc play "${WORKFLOW_NAME}" --no-detach
 workflow_run_ok "${TEST_NAME_BASE}-restart" \
     cylc play "${WORKFLOW_NAME}" --no-detach --reference-test
-contains_ok "${WORKFLOW_RUN_DIR}/log/job/1/foo/NN/job-activity.log" <<'__LOG__'
-[(('event-handler-00', 'execution timeout'), 1) cmd] echo 1/foo 'execution timeout'
-[(('event-handler-00', 'execution timeout'), 1) ret_code] 0
-[(('event-handler-00', 'execution timeout'), 1) out] 1/foo execution timeout
-__LOG__
```

Comment:
Why is this removed from the test?

Reply:
Because it no longer gets logged. According to @oliver-sanders that's OK.